lib/utils/analytics/generatePayload.js

```diff
@@ -85,5 +93,11 @@ module.exports = async (serverless) => {
     npmDependencies,
     triggeredDeprecations: Array.from(triggeredDeprecations),
     versions,
+    createdAt: Date.now(),
+    ciName: ci.name,
+    frameworkId: userConfig.get('frameworkId'),
+    userId: userConfig.get('userId'),
+    orgUid: serviceConfig.orgUid,
```

Comment:
In context of this payload we practice nested configuration, and we already have dashboard related property.

So maybe we can replace `isDashboardEnabled: true` with `dashboard: { userId: "xxx", orgUid: "xxx" }`

Such changes to payload are not harmful as long as in first place metrics service is updated and deployed and after that new version of framework with update published

Reply:
I didn't think of that but I'm wondering if it's a good approach here - `userId` can be available even if `isDashboardEnabled` is set to false, which will then result in inaccurate reporting of `isDashboardEnabled` flag. What do you think?

Reply:
Technically, it's true, that user may be logged in, but deploying the service which is not submitted to dashboard.

I wonder whether there's a value in having userId being send in such case, and weather it will not confuse our analytics. We definitely then should not assume that number of deployments with users logged into dashboard equals actual dashboard deployments.

Anyway, it's a problem we may tackle on our backend side, and here just send `dashboard: { userId, orgUid }`. Having that, it's `orgUid` which will inform on whether dashboard was enabled for given deployment (so still I think we don't need `isEnabled` setting)

Reply:
That makes perfect sense :+1:

Reply:
changed :+1: